docker/ingestor/http-sinks.toml

```diff
@@ -1,7 +1,18 @@
+[sinks.go_parser]
+type = "http"
+uri = "http://go-parser:7777"
+inputs = ["go_spans"] # only send spans for now
+compression = "gzip"
+encoding.codec = "json"
+batch.max_bytes = 10485760 # 10MB, required for rrweb payloads 
```

Comment:
Do we want rrweb spans? They should never have a db.statement right? Just double checking

Reply:
yeah, it was copy-n-paste. no big deal at this moment